tools/llm_bench/llm_bench_utils/model_utils.py

```diff
@@ -10,6 +10,20 @@
 import librosa
 
 
+KNOWN_PRECISIONS = [
```

Comment:
not sure if this function still in use, but can we reuse precisions mapping here https://github.com/openvinotoolkit/openvino.genai/blob/master/tools/llm_bench/llm_bench_utils/model_utils.py#L272